README.md

```diff
@@ -1,26 +1,42 @@
 # React Native Releases
 
-
-This repository is dedicated to coordinating React Native releases; if you want to learn more about *how* a release is worked on, please refer to the [releases documentation](https://reactnative.dev/contributing/release-roles-responsibilities).
+This repository is dedicated to coordinating React Native releases; if you want to learn more about _how_ a release is worked on, please refer to the [releases documentation](https://reactnative.dev/contributing/release-roles-responsibilities).
 
 ## Releases Support Policy
 
 ### Which versions are currently supported?
 
-We are supporting the **latest version**, and the **two previous minor series.**
-We also work on the **next version** being developed, which will become the new stable after its 0.Y.0 release.
+| Version  | Type                       | Support level |
+| -------- | -------------------------- | ------------- |
+| 0.73.x   | Next version               | Future        |
+| 0.72.x   | Latest stable              | Active        |
+| 0.71.x   | Previous (-1) minor series | Active        |
+| 0.70.x   | Previous (-2) minor series | End of Cycle  |
+| <=0.69.x | Old minor series           | Unsupported   |
 
-Current versions supported:
+### What level of support can be expected?
 
-| Version    | Type                  | Status           |
-| ---------- | --------------------- | ---------------- |
-| 0.73.x     | Next version          | Not started      |
-| 0.72.x     | Latest stable         | In support       |
-| 0.71.x     | Previous minor series | In support       |
-| 0.70.x     | Previous minor series | In support       |
-| <=0.69.x   | Old minor series      | Unsupported      |
+To set clear expectations and provide a sustainable upgrading experience for the community, the level of support provided by the release crew to the various versions of React Native is weighted to focus on the newer versions, while maintaining the previous two minors.
 
-### What level of support can be expected?
+The different statuses presented in the table are defined as such:
+
+* `Future`
+
+After a [new version branch gets cut](https://reactnative.dev/contributing/release-branch-cut-and-rc0), creating new Release Candidates to allow the community to test the upcoming version is very important. New RC releases are done at a high pace, as soon as viable.
+
+* `Active`
+
+Stable releases in active support receive frequent updates, up to a few every month. Latest stable has the highest priority, and at the start of its stable cycle (right after [.0 is released](https://reactnative.dev/contributing/release-stable-minor)) multiple patches will be done as soon as possible to stabilize the version and ensure a good upgrade experience to the community.
```

Comment:
I don't think "up to a few" every month is high frequency. Especially for the latest minor.

Reply:
I think one point worth clarifying: I think that there are two elements here:
* how frequent releases can be
* how much work one release takes

and you are suggesting that "high frequency" -> "a lot of work" because of the 2nd point. Fixing the second point (via better tooling, etc.) will resolve that. The frequency of the releases is the wrong element to focus on to tackle the "a lot of work" problem.